raiden/network/rpc/client.py

```diff
@@ -438,6 +439,31 @@ class JSONRPCClient(object):
         nonce_offset (int): Network's default base nonce number.
     """
 
+    def _checkNodeConnection(func, *args, **kwargs):
+        def retry_waits():
```

Comment:
I think this should reuse the [timeout_exponential_backoff](https://github.com/raiden-network/raiden/blob/08b8c819db1ac47ba4a9fe7378521751cd35f3be/raiden/network/protocol.py#L85) generator (better move it outside that module)